ci/synth_vivado.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Command line arguments
+
+# Default to 1 core
+N=${1:-1}
+
+# Bash array to iterate over for configurations
+cfgs=(\
+    "e_bp_unicore_cfg"
+    "e_bp_multicore_1_cfg"
+    "e_bp_multicore_1_cce_ucode_cfg"
+    "e_bp_multicore_4_cfg"
```

Comment:
Does 4 cores fit in whatever part you're targeting? 2 is probably more realistic?

What's the runtime of these?

Reply:
Yeah, 4 just went over, changed it to 2. It took about 10 minutes to run.